tests/invalid_models_tests/test_ordinary_fields.py

```diff
@@ -1057,3 +1058,69 @@ class Meta:
 
         errors = Model._meta.get_field("field").check(databases=self.databases)
         self.assertEqual(errors, [])
+
+
+class InvalidDBDefaultTests(TestCase):
```

Comment:
```suggestion
@isolate_apps("invalid_models_tests")
class InvalidDBDefaultTests(TestCase):
```
With `isolate_apps` you will be able to use the same model name in all tests e.g. `class Model(models.Model)`.